docs/register-phase-1-stake.md

```diff
@@ -0,0 +1,104 @@
+# Register your Phase-1 stake to Phase-2 through CLI
+
+## Understanding the registration process
+
+The registration of Phase-1 stakes to Phase-2 is a manual process that requires 
+the staker to register their stake to Phase-2 of the Babylon network. 
```

Comment:
I think we should avoid overly using the "Phase-1" and "Phase-2" concepts as they might be hard to understand. If you see the document I wrote on the registration eligibility, I note about the "registration of stakes that are confirmed on the Bitcoin ledger (such as those created during phase-1) to the Babylon blockchain".

Essentially, I refer to "phase-2" as the Babylon blockchain and I explain that phase-1 stakes are a subset of stakes already created on bitcoin